test/functional/sqlsrv/sqlsrv_encrypted_query_nosqltype.phpt

```diff
@@ -0,0 +1,138 @@
+--TEST--
+Test using sqlserv_query for binding parameters with ColumnEncryption enabled and a custome keystore provider
+--SKIPIF--
+<?php require('skipif_server_old.inc'); ?>
```

Comment:
if using a custom keystore provider, you will need to the other skipif specific for ksp
also, please fix the typo _custome_